specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,764 @@
+    AddButton(Button(L"Modify app settings").
+        SetInputId(L"textBox").
+        SetIsContextMenuPlacement(true))
+    .Show();
+```
+
+Example result:
+![Button ContextMenu Example](ButtonContextMenuExample.png)
+
+# Image
+
+The Image component API sets up the xml for the \<image\> element. Images are visual elements that
+are used to enhance an AppNotification.
+
+The image source can be pulled using one of these protocol handlers:
+
+-   http:// or https:// (A web-based image)
+-   ms-appx:/// (An image included in the app package)
+-   ms-appdata:///local/ (An image saved to local storage)
+-   file:/// (A local image)
+
+**WinAppSDK 1.2 \<image\> Schema**:
+
+```c#
+<image src = string
+       alt? = string
+       placement? = "appLogoOverride" | "hero"
+       hint-crop? = "circle" />
+```
+
+These attributes are abstracted with the Button component APIs:
+
+-   Image(String imageSrc)
+-   SetAlternateText(String alternateText)
+-   SetImagePlacement(ImagePlacement placement)
+-   SetUsesCircleCrop(Boolean usesCircleCrop)
+
+Images can be displayed in three different ways:
+
+## Inline
+
+A full-width inline-image that appears when you expand the AppNotification.
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"Below is an inline image"))
+    .AddImage(Image(LR"(https://unsplash.it/360/202?image=883)")
+        .SetImagePlacement(ImagePlacement::Inline)
+        .SetAlternateText(L"Image with nature"))
+    .Show();
+```
+
+Example result:
+![Inline Image Example](InlineImageExample.png)
+
+## AppLogoOverride
+
+Overrides the AppNotification app logo with a custom image.
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"To the left is an AppLogoOverride"))
+    .AddImage(Image(LR"(https://unsplash.it/64?image=669)")
+        .SetImagePlacement(ImagePlacement::AppLogoOverride)
+        .SetAlternateText(L"Profile picture"))
+    Show();
+```
+
+Example result:
+![AppLogoOverride Image Example](AppLogoOverrideImageExample.png)
+
+## Hero
+
+Prominently displays image within the AppNotification banner and while inside Notification Center.
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"Above is a Hero image"))
+    .AddImage(Image(LR"(https://unsplash.it/360/180?image=1043)")
+        .SetImagePlacement(ImagePlacement::Hero)
+        .SetAlternateText(L"Image with nature"))
+    .Show();
+```
+
+Example result:
+![Hero Image Example](HeroImageExample.png)
+
+For both Inline and AppLogoOverride, developers can set how the image will be cropped with
+SetUsesCircleCrop(bool).
+
+## Cropped AppLogoOverride
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"To the left is an AppLogoOverride"))
+    .AddImage(Image(LR"(https://unsplash.it/64?image=669)")
+        .SetImagePlacement(ImagePlacement::AppLogoOverride)
+        .SetUsesCircleCrop(true)
+        .SetAlternateText(L"Profile picture"))
+    .Show();
+```
+
+Example result:
+![Cropped AppLogoOverride Image Example](CroppedAppLogoOverrideExample.png)
```

Comment:
Your .AddText(Text(L"To the left is an AppLogoOverride")) above should be  .AddText(Text(L"Cropped AppLogoOverride")) to align with what is shown in image